CONTRIBUTING.md

```diff
@@ -8,48 +8,48 @@ Welcome to [TheAlgorithms/C-Plus-Plus](https://github.com/TheAlgorithms/C-Plus-P
 
 ### Contributor
 
-We are very happy that you consider implementing algorithms and data structures for others! This repository is referred to and used by learners from around the globe. Being one of our contributors, you agree and confirm that:
+We are happy that you consider implementing algorithms and data structures for others! This repository is referred to and used by learners from around the globe. Being one of our contributors, you agree and confirm that:
 
 - You did your own work.
   - No plagiarism allowed. Any plagiarized work will not be merged.
```

Comment:
- Please try to provide your own implementations with proper documentation. We do not entertain any plagiarized work.

Reply:
I don't think this is needed. The current paragraph is OK IMO.